### PR TITLE
Feature/support multi tenant chat

### PR DESCRIPTION
**Pull request by niztal**
In continuous to the conversation upon [https://github.com/FlowiseAI/Flowise/pull/1709](url) 
I've added the support of multi tenant chatId with concatenation of {customerId}_{chatId}.

This way while singe chatflow serves multiple tenants, we can still have grouping of chat messages by specific customer (later on also chat messages feedbacks) by querying all the rows which contains in the chatId column the required customer's id.

@chungyau97 @HenryHengZJ please review

Thanks

### CODE REVIEW

**Comment by niztal** [comment]
@HenryHengZJ can you please review?

File: playground/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import express from 'express';
```
Comment (HenryHengZJ) [inline playground/index.js:1]: can we remove the playground as we have created another playground here as too - https://github.com/FlowiseAI/Flowise/pull/1639

File: src/components/Bot.tsx [hunk 2]
```diff
@@ -182,7 +182,9 @@ export const Bot = (botProps: BotProps & { class?: string }) => {
   );
   const [socketIOClientId, setSocketIOClientId] = createSignal('');
   const [isChatFlowAvailableToStream, setIsChatFlowAvailableToStream] = createSignal(false);
-  const [chatId, setChatId] = createSignal(uuidv4());
+  const [chatId, setChatId] = createSignal(
+    props.chatflowConfig?.customerId ? `${props.chatflowConfig?.customerId.toString()}_${uuidv4()}` : uuidv4(),
```
[the remark this answers is not in the document]
Comment (HenryHengZJ) [reply src/components/Bot.tsx:186]: image

File: playground/public/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<html>
```
Comment (HenryHengZJ) [inline playground/public/index.html:1]: can we remove this as well

**Comment by niztal** [comment]
@HenryHengZJ I've fixed all of your comments, please review

Thanks

File: pnpm-lock.yaml [hunk 4]
```diff
@@ -0,0 +1,4570 @@
+lockfileVersion: '6.0'
```
Comment (HenryHengZJ) [inline pnpm-lock.yaml:1]: sorry this file was accidentally merged last time. it has been removed, can we remove from this PR as well

Comment (niztal) [reply pnpm-lock.yaml]: sure @HenryHengZJ , done
please review

**Review by HenryHengZJ — APPROVED** [review]

**Comment by HenryHengZJ** [comment]
thanks @niztal !

**Pull request merged**